### DISCUSSION TITLE
InfinityFlow.Aspire.Temporal 0.5.8

### DISCUSSION THREAD

**Discussion by ElanHasson** [InfinityFlowApp/aspire-temporal#74] [Announcements]
- chore(deps): bump Aspire.Hosting.AppHost from 8.0.0-preview.7.24251.11 to 8.0.1 in the aspire group by @dependabot (#71)
- chore(deps): bump Microsoft.Extensions.ServiceDiscovery from 8.0.0-preview.7.24251.11 to 8.0.1 in the dotnet group by @dependabot (#72)
- chore(deps): bump OpenTelemetry.Instrumentation.Runtime from 1.8.0 to 1.8.1 in the opentelemetry group by @dependabot (#73)

## 👨🏼‍💻 Contributors

@dependabot and @dependabot[bot]

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityFlowApp/aspire-temporal/releases/tag/0.5.8'>InfinityFlow.Aspire.Temporal 0.5.8</a>.</em>

**Comment by DavidC2509** [comment]
Good afternoon, nice to meet you, I am a .NetAspire C# developer and I was implementing InfinityFlow and everything was fine locally. I had a question, let's say I want to upload the project to Azure. I think the "aspire-temporal-server-helper" image is not yet public. And I would like to ask you how would the upload of InfinityFlow to Azure be with NetAspire? Or do you recommend using TemporalCloud?

**Reply by ElanHasson** [reply]
👋🏼 the image is built dynamically on startup.

Are you having an issue?
